### PR TITLE
FQDN up to 25 characters

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### :nut_and_bolt: Description: What code changed, and why?

Define the length of the FQDN in the configurations doc.
Includes some linting cleanup.

### :chains: Related Resources

#4890 

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [x] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/configuration.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ This section describes those settings and how to change them on an existing Chef
 
 #### Chef Automate FQDN
 
-To change the fully qualified domain name (FQDN) of your Chef Automate installation, create a TOML file that contains the partial configuration:
+The maximum size of a customized fully qualified domain name (FQDN) is 25 characters. To change the  of your Chef Automate installation, create a TOML file that contains the partial configuration:
```
Comment (msorens) [inline components/docs-chef-io/content/automate/configuration.md:34]: Lost a subject in there.
```suggestion
The maximum size of a customized fully qualified domain name (FQDN) is 25 characters. To change the FQDN of your Chef Automate installation, create a TOML file that contains the partial configuration:
```

Comment (msorens) [inline components/docs-chef-io/content/automate/configuration.md:34]: Good guess, but, well, not so much. There is no limit on the size of the whole FQDN string.
The 25-character limit is only _after_ the final dot, so something like this is invalid:
`http://www.foo.this-portion-is-probably-too-long`

**Review by SEAjamieD — APPROVED** [review]
Pending and update for @msorens comment below, this is good for our needs.  Thanks so much!

>There is no limit on the size of the whole FQDN string.
The 25-character limit is only after the final dot, so something like this is invalid:
http://www.foo.this-portion-is-probably-too-long

File: components/docs-chef-io/content/automate/configuration.md (hunk 1 above)
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/configuration.md:34]: @SEAjamieD So 25 characters for the top-level domain?

http://www.foo.this-portion-is-probably-too-long
<scheme>://<subdomain>.<second-level domain>.<top-level domain>

Comment (SEAjamieD) [reply components/docs-chef-io/content/automate/configuration.md]: @kagarmoe - Correct, thank you!

**Pull request merged**